config/environments/production.rb

```diff
@@ -43,7 +43,7 @@
   # config.force_ssl = true
 
   # Set to :debug to see everything in the log.
-  config.log_level = :info
+  config.log_level = :debug
```

Comment:
If we don't run epp exceptions with debug severity there is no good reason to change from info to debug

Reply:
It is a new default https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels. What problem you see here?

Reply:
because you didnot reason the change i had wrong assumptions if thats new default its okey with me.

Reply:
My main point was still that it might be good idea to log everything, with no regard to performance penalty (until it becomes a problem which can be measured), that's why I asked if you see any problem here. We can (and should) tune the software so that it works the way we want, not we work the way our software does.

P.S. Of coerce the effort to deviate from the defaults should be taken into account, but in this case it's just a matter of turning one single config on or off.